librz/core/disasm.c

```diff
@@ -5164,7 +5164,7 @@ RZ_API void rz_analysis_disasm_text_free(RzAnalysisDisasmText *t) {
  */
 RZ_API int rz_core_print_disasm(RZ_NONNULL RzCore *core, ut64 addr, RZ_NONNULL ut8 *buf, int len, int nlines, RZ_NULLABLE RzCmdStateOutput *state,
```

Comment:
Update Doxygen comment that len and lines can't be zero, also update/remove relevant tests

Reply:
If we prohibit zero here, `pd`/`pi`/etc also should print some error message for prohibited values.